### ISSUE TITLE
Training always starts with accuracy of 1.000

### ISSUE THREAD

**Issue by gopinath-balu** [bfortuner/pytorch_tiramisu#22]
Thank you so much @bfortuner for the implementation of this model in PyTorch. When I try to train the Network with the mentioned CamVid dataset, my training always starts with the Train Acc and Val Acc of 1.000. I have set the hyper-parameters as mentioned in the requirements section. 
![Screenshot 2019-08-30 at 6 31 23 PM](https://user-images.githubusercontent.com/12084646/64023029-136bc780-cb55-11e9-8437-6cd98536acee.png)

In other words training error and validation error is always 0. Wondering if am missing something ??

**Comment by KerryZack** [comment]
maybe you can check the err function in the training.py.I replace the "return round(err,5)" with return err.item()
